packages/analytics-js-plugins/__tests__/deviceModeDestinations/utils.test.ts

```
@@ -262,12 +214,131 @@ describe('deviceModeDestinations utils', () => {
+          },
+        },
+      };
+
+      expect(isDestinationSDKMounted(destSDKIdentifier, sdkTypeName)).toEqual(false);
+    });
+
     it('should return true if the destination SDK is a constructable type', () => {
       (window as any)[destSDKIdentifier] = {
         [sdkTypeName]: class {
+          // eslint-disable-next-line sonarjs/no-useless-constructor, @typescript-eslint/no-useless-constructor, sonarjs/no-empty-function, @typescript-eslint/no-empty-function
           constructor() {}
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)